File: Modul 4/Bahasa C/PRAK401-2310817120001-DinaIzzatiElfadheya.c
#include <stdio.h>

int main() {
    int bilangan;
    char simbol;

    printf("Input\n");
    scanf("%d %c", &bilangan,  &simbol);
    printf("\nOutput\n");

    for (int i = 1; i <= 50; i++) { //loop 1-50
        if (i % bilangan == 0) {
            printf("%c ", simbol);
        } else {
            printf("%d ", i);
        }
    }

    return 0;
}

File: Modul 4/Bahasa Python/PRAK401-2310817120001-DinaIzzatiElfadheya.py
print("Input")
input_str = input().split()
bilangan = int(input_str[0])
simbol = input_str[1]

print("\nOutput")

for i in range(1, 51):  # loop 1-50
    if i % bilangan == 0:
        print(simbol, end=" ")
    else:
        print(i, end=" ")